syntax = "proto3";

package ExServer;

service ExServiceAPI
{
  rpc unaryRPC (RequestMsgEx) returns (ResponseMsgEx) {}
  rpc serverStreamingRPC (RequestMsgEx) returns (stream ResponseMsgEx) {}
  rpc clientStreamingRPC (stream RequestMsgEx) returns (ResponseMsgEx) {}
  rpc bidirectionalStreamingRPC (stream RequestMsgEx) returns (stream ResponseMsgEx) {}
}

message RequestMsgEx {
  string foo = 1;
}

message ResponseMsgEx {
  string bar = 1;
}
